pep/app/authenticators/credentials.ts

```diff
@@ -0,0 +1,164 @@
+import { inject as service } from '@ember/service';
+import { isEmpty } from '@ember/utils';
+import { resolve, reject } from 'rsvp';
+import BaseAuthenticator from 'ember-simple-auth/authenticators/base';
+
+import AjaxService from 'pep/services/ajax';
+import ENV from 'pep/config/environment';
+import { PepSecureAuthenticatedData } from 'pep/api';
+import { serializeQueryParams } from 'pep/utils/url';
+import PepSessionService from 'pep/services/pep-session';
+import { run } from '@ember/runloop';
+import { EmberRunTimer } from '@ember/runloop/types';
+import FastbootService from 'ember-cli-fastboot/services/fastboot';
+
+export enum SessionType {
```

Comment:
thoughts on defining this in types/pep/api.d.ts? since both authenticators really "own" it equally.